Sources/ContentsquareInstance.swift

```diff
@@ -51,6 +53,37 @@ public class ContentsquareInstance: ContentsquareCommand {
         }
     }
     
+    public func sendUserIdentifier(userId: String) {
+        Contentsquare.sendUserIdentifier(userId)
+    }
+    
+    public func sendCustomVars(screenName: String, customVars: [[String: Any]]) {
+        let csCustomVars = customVars.compactMap { json -> CustomVar? in
+            guard let index = parseIndex(json["index"]) else {
+                print("CustomVar index not convertible: \(json)")
+                return nil
+            }
+            
+            guard index > 0 && index <= 20 else {
+                print("CustomVar index out of range (1-20): \(index)")
+                return nil
+            }
+            
+            guard let name = json["name"] as? String, !name.isEmpty,
+                  let value = json["value"] as? String, !value.isEmpty else {
```

Comment:
It looks like empty name and value are allowed (and internally replaced with `cs-empty`)

```suggestion
            guard let name = json["name"] as? String,
                  let value = json["value"] as? String else {
```